### ISSUE TITLE
add rasusa 0.5.0

### ISSUE THREAD

**Issue by kapsakcj** [StaPH-B/docker-builds#250]
https://github.com/mbhall88/rasusa/releases/tag/0.5.0

**Comment by mbhall88** [comment]
Being a bit of a github creep here... But my [Docker recipe on rasusa](https://github.com/mbhall88/rasusa/blob/master/Dockerfile) ends up coming in a lot smaller ([~6MB](https://quay.io/repository/mbhall88/rasusa?tab=tags)) than this recipe (which seems to be about 31MB) due to the use of multi-stage build and no underlying ubuntu OS.

Not sure if container size is of any interest to you, but feel free to "steal" my Dockerfile if you like.

**Comment by kapsakcj** [comment]
>Being a bit of a github creep here...

Not at all! 😆 Glad you found your way here. I'm a big fan of `rasusa`, awesome work on the tool.

I'm sure it is useful to the folks that strive for super lean containers, but I'm not sure it's necessary to re-host/duplicate effort, as it looks like you have it setup to automatically build images on quay. Appreciate the offer though!
